src/test/java/uk/gov/hmcts/reform/civil/utils/HearingUtilsTest.java

```diff
@@ -63,20 +67,55 @@ void shouldReturnFee7_whenGivenAnyClaimFee() {
     }
 
     @Test
-    void shouldReturnFormattedFee_whenGivenAnyClaimFee() {
+    void shouldReturnFormattedFee_whenGivenMidClaimFee() {
         assertThat(HearingUtils.formatHearingFee(
             Fee.builder().calculatedAmountInPence(new BigDecimal(34600)).build())).isEqualTo("£346");
     }
 
     @Test
-    void shouldReturnFormattedFee2_whenGivenAnyClaimFee() {
+    void shouldReturnFormattedFee_whenGivenBigClaimFee() {
         assertThat(HearingUtils.formatHearingFee(
             Fee.builder().calculatedAmountInPence(new BigDecimal(132000)).build())).isEqualTo("£1,320");
     }
 
     @Test
-    void shouldReturnFormattedFee3_whenGivenAnyClaimFee() {
+    void shouldReturnFormattedFee_whenGivenSmallClaimFee() {
         assertThat(HearingUtils.formatHearingFee(
             Fee.builder().calculatedAmountInPence(new BigDecimal(5000)).build())).isEqualTo("£50");
     }
+
+    @Test
+    void shouldReturnNull_when0ClaimFee() {
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(new BigDecimal(0)).build())).isNull();
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {34600, 132000, 5000})
+    void shouldReturnFormattedFee_whenGivenAnyClaimFee(int amount) {
+        BigDecimal feeAmount = new BigDecimal(amount);
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(feeAmount).build())).isNotEmpty();
+    }
```

Comment:
I feel we lost something in this change. Before, with the three separate tests, we were checking for the output correctness, now we don't anymore. That's why even though you did add a `@ParameterizedTest` as I asked, you also had to leave the previous tests.

Consider the following instead:
```
    @ParameterizedTest
    @CsvSource(value = {"34600;£346", "132000;£1,320", "5000;£50"}, delimiter=';')
    void shouldReturnFormattedFee_whenGivenAnyClaimFee(int amount, String expectedOutput) {
        BigDecimal feeAmount = new BigDecimal(amount);
        assertThat(HearingUtils.formatHearingFee(
            Fee.builder().calculatedAmountInPence(feeAmount).build())).isEqualTo(expectedOutput);
    }
```

The above should check the correctness of the output both in terms of "not empty" and in terms of "formatted fee".

This should make the `shouldReturnFormattedFee_whenGivenMidClaimFee`, `shouldReturnFormattedFee_whenGivenBigClaimFee` and `shouldReturnFormattedFee_whenGivenSmallClaimFee` tests no longer needed.